internal/dinosaur/constants/cluster.go

```diff
@@ -10,9 +10,32 @@ const (
 	// ClusterOperationDelete - OpenShift/k8s cluster delete operation
 	ClusterOperationDelete ClusterOperation = "delete"
 
+	// DefaultIngressDnsNamePrefix the default name prefix of the default ingress dns
+	DefaultIngressDnsNamePrefix = "apps"
+)
+
+// TODO change these constants to match your own
+const (
+	// DinosaurOperatorAddonNamespace is the namespace where the Operator is deployed on via the AddOn flow in Production OSD clusters
+	DinosaurOperatorAddonNamespace = "redhat-managed-dinosaur-operator"
+	// DinosaurOperatorQEAddonNamespace is the namespace where the Operator is deployed on via the AddOn flow in Stage OSD clusters
+	DinosaurOperatorQEAddonNamespace = "redhat-managed-dinosaur-operator-qe"
+	// FleetshardAddonNamespace is the namespace where the fleetshard Operator is deployed on via the AddOn flow in Production OSD clusters
+	FleetshardAddonNamespace = "redhat-fleetshard-operator"
+	// FleetshardQEAddonNamespace is the namespace where the fleetshard Operator is deployed on via the AddOn flow in Stage OSD cluster
+	FleetshardQEAddonNamespace = "redhat-fleetshard-operator-qe"
+	// OpenIDIdentityProviderName is the name of the Identity Provider configured for each OSD cluster
+	OpenIDIdentityProviderName = "Dinosaur_SRE"
+	// ReadOnlyGroupName is the group name for users with read only access
+	ReadOnlyGroupName = "dinosaur-readonly-access"
+	// SREGroupName is the group name contain users with cluster admin role
+	SREGroupName = "dinosaur-sre"
+	// ReadOnlyRoleBindingName is the read only binding name
```

Comment:
Same for ReadOnlyRoleBindingName and SRERoleBindingName. It would be good to be explicit about this being K8s-related concepts

Reply:
updated